modules/seqexec/server/src/main/scala/seqexec/server/keywords/StandardHeader.scala

```diff
@@ -102,11 +102,11 @@ class StandardHeader[A: HeaderProvider](
     buildString(obsReader.getObsType, "OBSTYPE"),
     buildString(obsReader.getObsClass, "OBSCLASS"),
     buildString(obsReader.getGemPrgId, "GEMPRGID"),
-    buildString(obsReader.getObsId, "obsid"),
+    buildString(obsReader.getObsId, "OBSID"),
     buildString(obsReader.getDataLabel, "DATALAB"),
     buildString(stateReader.getObserverName, "OBSERVER"),
     buildString(obsReader.getObservatory, "OBSERVAT"),
-    buildString(obsReader.getTelescope, "telescope"),
+    buildString(obsReader.getTelescope, "TELESCOP"),
```

Comment:
`telescope` is not a valid keyword name. I'll do a refactoring of the keywords code to make this impossible

Reply:
`telescope` is valid in the DHS.

Reply:
Would it work regardless with the keyword in capitals? Let me review why GPI uses this:

`TELESCOP= 'Gemini-South'       / Name of telescope (Gemini-North|Gemini-South)`